### ISSUE TITLE
Infinite List doesn't have an indicator when using Keyboard Navigation

### ISSUE THREAD

**Issue by JakeLaCombe** [cerner/terra-framework#261]
# Bug Report

## Description
When a user is scrolling down an infinite list with a keyboard, there is no indication of when the user has reached the end of the list. The list still generates more items, but doesn't tell the user about it. Should the list have an indication of some kind to tell those on Screen Readers more items are being generated?

**Comment by bjankord** [comment]
We've talked about using aria-live to indicate more items have been added to the list. Need to determine what the verbiage for this message should be and on what events it should occur.

**Comment by bjankord** [comment]
PR: https://github.com/cerner/terra-framework/pull/732
